routing/cross_mwm_road_graph.cpp

```diff
@@ -92,36 +92,40 @@ class EdgesFiller
 };
 
 bool ForEachNodeNearPoint(CrossRoutingContextReader const & currentContext,
-                          CrossNode const & crossNode,
+                          ms::LatLon const & point,
                           ClosestNodeFinder<IngoingCrossNode> const & findingNode)
 {
-  return currentContext.ForEachIngoingNodeNearPoint(crossNode.point, findingNode);
+  return currentContext.ForEachIngoingNodeNearPoint(point, findingNode);
 }
 
 bool ForEachNodeNearPoint(CrossRoutingContextReader const & currentContext,
-                          CrossNode const & crossNode,
+                          ms::LatLon const & point,
                           ClosestNodeFinder<OutgoingCrossNode> const & findingNode)
 {
-  return currentContext.ForEachOutgoingNodeNearPoint(crossNode.point, findingNode);
+  return currentContext.ForEachOutgoingNodeNearPoint(point, findingNode);
 }
 
 template <class Node>
-void FindCrossNode(CrossRoutingContextReader const & currentContext, CrossNode const & crossNode,
+bool FindCrossNode(CrossRoutingContextReader const & currentContext, CrossNode const & crossNode,
                    Node & node)
 {
   double minDistance = std::numeric_limits<double>::max();
   ClosestNodeFinder<Node> findingNode(crossNode, minDistance, node);
-  CHECK(ForEachNodeNearPoint(currentContext, crossNode, findingNode), ());
-  CHECK_NOT_EQUAL(minDistance, std::numeric_limits<double>::max(),
-                  ("crossNode.point:", crossNode.point));
+  CHECK(ForEachNodeNearPoint(currentContext, crossNode.point, findingNode), ());
+  if (minDistance == std::numeric_limits<double>::max())
```

Comment:
Please, create a named constant for double max, say, kInvalidDistance, or kInfinity or something like that.